posters-overrides.json

```diff
@@ -4170,7 +4170,13 @@
    "number": 1481
   },
   {
-   "number": 1482
+   "number": 1482,
+   "title": "Understanding deep learning-based brain age predictions",
+   "institution": "Max Planck Institute for Human Cognitive and Brain Sciences",
+   "presenter": "Simon Hofmann",
+   "categories": "Modeling and Analysis Methods<br>Classification and Predictive Modeling",
+   "videochat": "<a href=\"https://meet.jit.si/ohbm2020-1482\" target=\"_ohbm2020-1482\">jitsi:ohbm2020-1482</a>",
+   "pdf": "<a https://owncloud.gwdg.de/index.php/apps/onlyoffice/s/SxcfCjF1d1QJAm2</a>"
```

Comment:
pleast list only what eds to be changed, not full entry

also pdf -- should be just a url, remove all the `<a` -- see other entries in the file

Reply:
Alright thanks. I send my new pull request now